docs/src/modules/getting-started/pages/author-your-first-service.adoc

```diff
@@ -182,6 +170,71 @@ curl -i -XPOST -H "Content-Type: application/json" localhost:9000/hello -d '
 
 Does it recognize the pirate greeting?
 
+== Play a language game
+
+In this section, you'll play a simple language game with the agent using `curl`.
+
+Each player must reply in a new language. The agent remembers previous languages and will end the game if a language is repeated.
+
+To keep the game predictable, the agent is instructed to always begin in English. This prevents accidental repetition and allows you to observe memory in action.
+
+[NOTE]
+====
+Be sure to include the language you're using in parentheses at the end of your message. This helps the agent track the game state correctly.
+====
+
+. Start the game:
+
+[source, command line]
+----
+curl -i -XPOST -H "Content-Type: application/json" localhost:9000/hello -d \
+'{"greeting":"Let'\''s play a language game. Each of us must reply in a new language every time. The first to repeat a language loses. Try to not lose the game. Always name the language you'\''re using in parentheses. You start. Use English first. If I repeat a language, announce that the game is over and that you won.","sessionId":"12345"}'
+----
```

Comment:
I thought we would completely replace the hello world with the language game. This would the system message of the agent. Now we are kind of abusing the HelloWorldAgent to do something it was not intended to do.

If we decide to only use the language game we would also change the archetype so that it generates the LanguageGameEndpoint and LanguageGameAgent.

Also note that the POST was an addition made by the user in previous step, so it's not possible to play the game without adding that code.